cmake/Dependencies.common.cmake

```diff
@@ -66,6 +66,19 @@ else()
   message(STATUS "Building WITHOUT JpegTurbo")
 endif()
 
+##################################################################
+# libtiff
+##################################################################
+if (BUILD_LIBTIFF)
+  find_package(TIFF REQUIRED)
+  include_directories(${TIFF_INCLUDE_DIR})
+  message("Using libtiff at ${TIFF_LIBRARY}")
+  list(APPEND DALI_LIBS ${TIFF_LIBRARY})
+  add_definitions(-DDALI_USE_LIBTIFF)
```

Comment:
It's already handled by a macro `propagate_option(BUILD_LIBTIFF)`, that will give you and `LIBTIFF_ENABLED` defined to 1 or 0 depending on the case. Please remove it, and use the generic solution.
```suggestion
```

Reply:
will do